tests/Frameworks/Symfony/Version_4_0/config/packages/validator.yaml

```diff
@@ -1,3 +1,2 @@
 framework:
-    validation:
```

Comment:
What was this for, and why has it changed?

Reply:
this is some internal validation module for symfony that we are not using. the configuration interface changed and given that we are not using it I think we should only use defaults